install/kubernetes/tetragon/values.yaml

```diff
@@ -175,6 +175,8 @@ tetragon:
     # -- The address at which to expose gRPC. Examples: localhost:54321, unix:///var/run/cilum/tetragon/tetragon.sock
     address: "localhost:54321"
   gops:
+     # -- Whether to enable exposing gops server.
+    enabled: false
```

Comment:
It will be user-facing change however, I imagine that the previous situation was making this enabled by default. However I think disabling it by default might be correct indeed!